v2/docs/gs/params.md

```diff
@@ -186,27 +213,55 @@ for additional details.
 #### Media Class Usage
 
 The media class usage policy for a KVS defines how the key-value data in
-that KVS is stored and managed in a KVDB configured with a staging media class.
+that KVS is stored and managed in a KVDB.
+Key and value data in a KVS can either be *pinned* to a particular media class,
+or *tiered* from one media class to another as it ages.
+This behavior is determined by the `mclass.policy` setting for the KVS.
 
-Key-value data in a KVS can either be *pinned* to a particular media class,
-or *tiered* from the staging media class to the capacity media class as it
-ages, as determined by the `mclass.policy` value for the KVS:
+The following `mclass.policy` settings pin key and value data:
 
-* `capacity_only` pins all key-value data to the capacity media class
-* `staging_only` pins all key-value data to the staging media class
-* `staging_min_capacity` tiers all key-value data
-* `staging_max_capacity` pins all key data to the staging media class and
-tiers all value data
+* `capacity_only` pins all key and value data to the capacity media class
+* `staging_only` pins all key and value data to the staging media class
+* `pmem_only` pins all key and value data to the pmem media class
 
-Of the two tiering options, `staging_max_capacity` will generally yield the
-highest throughput, lowest latency, and least write-amplification in the
-capacity media class.  The trade-off is more storage required in the
-staging media class.
+The following `mclass.policy` settings tier key and value data:
+
+* `staging_max_capacity` pins all key data to the staging media class and
+tiers all value data from staging to capacity
+* `staging_min_capacity` tiers all key and value data from the staging to
+the capacity media class
+* `pmem_max_capacity` pins all key data to the pmem media class and tiers all
+value data from pmem to capacity
+
+Below is a visualization of these `mclass.policy` settings in terms of
+which media classes may contain key and value data.
+
+| mclass.policy | pmem | staging | capacity |
+| :-- | :-- | :-- | :-- |
+| `capacity_only` | | | keys, values |
+| `staging_only` | | keys, values | |
+| `pmem_only` | keys, values | | |
+| `staging_max_capacity` | | keys, values | values |
+| `staging_min_capacity` | | keys, values | keys, values |
+| `pmem_max_capacity` | keys, values | | values |
 
 !!! info
-    If no staging media class is present, and an `mclass.policy` value other
-    than `capacity_only` is specified, a warning is logged and `capacity_only`
-    is applied.
+    Specifying an `mclass.policy` that references a media class
+    not configured for the KVDB will result in an error from the
+    `hse_kvdb_kvs_open()` API call.
```

Comment:
Helpful information.  But not necessary here.  Should be documented with the API in hse.h

Reply:
Yeah, it may be TMI.  I was trying to compensate for the fact that we removed the fallback behavior.